test/suites/standard/kustomize.robot

```diff
@@ -82,17 +84,18 @@ Setup Suite    # robocop: disable=too-long-keyword
     Set Suite Variable    \${ETC_SUBDIR_NAMESPACE}    ${ns}
 
     # The metal CI automation currently _only_ works for ostree deployments,
-    # and /usr/lib is not writable there.
-    #
-    # # Used by "Load From /usr/lib/microshift/manifests"
-    # ${ns}=    Generate Manifests    /usr/lib/microshift/manifests
-    # Set Suite Variable    \${USR_NAMESPACE}    ${ns}
-    #
-    # # Used by "Load From /usr/lib/microshift/manifestsd"
-    # ${rand}=    Generate Random String
-    # Set Suite Variable    \${USR_SUBDIR}    /usr/lib/microshift/manifests.d/${rand}
-    # ${ns}=    Generate Manifests    ${USR_SUBDIR}
-    # Set Suite Variable    \${USR_SUBDIR_NAMESPACE}    ${ns}
+    # and /usr/lib is not writable there, we use `Setup If OSTree Host` to detect
+    # an ostree filesystem and make /usr/lib temporarly writable, we restart during
+    # teardown to go back to initial state.
```

Comment:
Do we still need that comment? Would descriptive keyword name for usroverlay and description cover that?

Reply:
Yeah, maybe not as much needed now, made it more descriptive and removed comment.